ion_phys/rate_equations.py

```diff
@@ -59,11 +59,12 @@ def get_stim(self, lasers):
             delta_lu = Eu - El
 
             # Total scattering rate out of each state
-            GammaJ = GammaJ[upper_states]
-            GammaJ = np.repeat(GammaJ, n_lower).reshape(n_upper, n_lower).T
-            GammaJ2 = np.power(GammaJ, 2)
+            GammaJ_reduced = GammaJ[upper_states]
```

Comment:
nit pick: `_subspace` or similar would have been more in keeping with the notation used in the rest of the code.

Reply:
or just `_subs` to save characters.